partcad/tests/unit/test_git_clone_retries.py

```
@@ -15,123 +14,108 @@
-    GitCommandError(
-        f"""
-          ssh: Could not resolve hostname github.com: Name or service not known
-          fatal: Could not read from remote repository.
-        """
-    ),
+    GitError("failed to resolve address for github.com: Name or service not known"),
+    # The remote could not be reached
+    GitError("failed to connect to github.com: Connection refused"),
+    # Timeout, either the remote's own or the one PartCAD imposes
+    GitError("could not read from socket: timed out"),
+    GitError(f"failed to connect to github.com: Operation timed out"),
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)